stellargraph/connector/neo4j/mapper.py

```diff
@@ -72,6 +72,36 @@ def __init__(self, G, batch_size, neo4j_graphdb, schema=None):
         # Create Neo4J graph database object
         self.neo4j_graphdb = neo4j_graphdb
 
+    def flow(self, node_ids, targets=None, shuffle=False, seed=None):
```

Comment:
I added in a conversion from node ids to ilocs in `BatchedNodeGenerator` but, AFAIK Neo4j operates on node ids so I'm overloading `flow` here to avoid this conversion.